### PR TITLE
Micro speech build fix

### PR DESCRIPTION
**Pull request by plonajakub**
I've cleaned current project files. This is a working build, that has been tested in Renode by me. Feel free to ask me about anything that is unclear.

The  changes:
- one cmake file strategy has been obeyed
 - cmake has clear paths from now on
 - overlay and prj.conf are crucial for the project to work, so they have been added
 - command_responder.cc by @szymon-filipiak has been added to the target

### CODE REVIEW

**Review by Marzecki — APPROVED** [review]

File: tensorflow/lite/micro/examples/micro_speech/zephyr_riscv/CMakeLists.txt [hunk 1]
```diff
@@ -17,50 +27,123 @@ list(REMOVE_ITEM MICRO_SOURCES ${MICRO_TESTS_SOURCES})
...
+          ${MICRO_SPEECH}/main_functions.cc
+          ${MICRO_SPEECH}/main_functions.h
+          ${MICRO_SPEECH}/recognize_commands.cc
+          ${MICRO_SPEECH}/recognize_commands.h
+          ${MICRO_SPEECH}/no_1000ms_sample_data.cc
+          ${MICRO_SPEECH}/no_1000ms_sample_data.h
+          ${MICRO_SPEECH}/yes_1000ms_sample_data.cc
+          ${MICRO_SPEECH}/yes_1000ms_sample_data.h
+          # ${MICRO_SPEECH}/no_30ms_sample_data.cc
+          # ${MICRO_SPEECH}/no_30ms_sample_data.h
```
Comment (szymon-filipiak) [inline tensorflow/lite/micro/examples/micro_speech/zephyr_riscv/CMakeLists.txt:67]: What are those comments? Do we need this?

File: tensorflow/lite/micro/examples/micro_speech/zephyr_riscv/CMakeLists.txt [hunk 2]
```diff
@@ -17,50 +27,123 @@ list(REMOVE_ITEM MICRO_SOURCES ${MICRO_TESTS_SOURCES})
...
+          ${TF_LITE_DIR}/core/api/op_resolver.cc
+          ${TF_LITE_DIR}/core/api/tensor_utils.cc
+          ${TF_LITE_DIR}/kernels/internal/quantization_util.cc
+		  ${TF_LITE_DIR}/kernels/kernel_util.cc
+		  
+          # zephyr-src
+          src/audio_provider.cc
+          src/audio_provider.h
+          src/command_responder.cc
+		  src/command_responder.h
```
Comment (szymon-filipiak) [inline tensorflow/lite/micro/examples/micro_speech/zephyr_riscv/CMakeLists.txt]: Can we have a consistent indentation?

File: tensorflow/lite/micro/examples/micro_speech/zephyr_riscv/CMakeLists.txt [hunk 3]
```diff
@@ -17,50 +27,123 @@ list(REMOVE_ITEM MICRO_SOURCES ${MICRO_TESTS_SOURCES})
...
+          ${MICROFRONTEND}/frontend_util.h
+          ${MICROFRONTEND}/log_lut.h
+          ${MICROFRONTEND}/log_scale.h
+          ${MICROFRONTEND}/log_scale_util.h
+          ${MICROFRONTEND}/noise_reduction.h
+          ${MICROFRONTEND}/noise_reduction_util.h
+          ${MICROFRONTEND}/pcan_gain_control.h
+          ${MICROFRONTEND}/pcan_gain_control_util.h
+          ${MICROFRONTEND}/window.h
+		  ${MICROFRONTEND}/window_util.h
```
Comment (szymon-filipiak) [inline tensorflow/lite/micro/examples/micro_speech/zephyr_riscv/CMakeLists.txt]: Indentation ^^

File: tensorflow/lite/micro/examples/micro_speech/zephyr_riscv/prj.conf [hunk 4]
```diff
@@ -1,2 +1,3 @@
 CONFIG_CPLUSPLUS=y
 CONFIG_NEWLIB_LIBC=y
+CONFIG_ETH_LITEETH=n
```
Comment (zawadzki-dawid) [inline tensorflow/lite/micro/examples/micro_speech/zephyr_riscv/prj.conf:3]: What is the intended use of these line? Does it fix the problem with writing to ethernet we had earlier?

**Review by zawadzki-dawid — APPROVED** [review]
Looks good to me. Good job @plonajakub.

File: tensorflow/lite/micro/examples/micro_speech/zephyr_riscv/prj.conf (hunk 4 above)
Comment (szymon-filipiak) [reply tensorflow/lite/micro/examples/micro_speech/zephyr_riscv/prj.conf:3]: This line is disabling the ethernet support on the board. Yes, it was related to the issue we had earlier.

File: tensorflow/lite/micro/examples/micro_speech/zephyr_riscv/CMakeLists.txt (hunk 2 above)
Comment (plonajakub) [reply tensorflow/lite/micro/examples/micro_speech/zephyr_riscv/CMakeLists.txt]: Sure we can, but I am already tired of fight with vs code and cli editors to have it done :). Believe me or not but I've downloaded special plugin for proper formatting and it was looking good locally. Git seems to break it. If you have time pls try to do the formatting. Maybe I could make use of some git configuration tips :p. (I think the point is in \r\n - \n conversions.)

File: tensorflow/lite/micro/examples/micro_speech/zephyr_riscv/CMakeLists.txt (hunk 1 above)
Comment (plonajakub) [reply tensorflow/lite/micro/examples/micro_speech/zephyr_riscv/CMakeLists.txt:67]: Yup. We need them, it is still part of the micro_speech source or ready for use audio samples (for testing purposes). In order to have smaller binary I've commented them for now.

File: tensorflow/lite/micro/examples/micro_speech/zephyr_riscv/CMakeLists.txt (hunk 3 above)
Comment (plonajakub) [reply tensorflow/lite/micro/examples/micro_speech/zephyr_riscv/CMakeLists.txt]: Let's talk about it in the first conversation about indentation.

File: tensorflow/lite/micro/examples/micro_speech/zephyr_riscv/CMakeLists.txt (hunk 1 above)
Comment (plonajakub) [reply tensorflow/lite/micro/examples/micro_speech/zephyr_riscv/CMakeLists.txt:67]: Please resolve this if everything is clear

File: tensorflow/lite/micro/examples/micro_speech/zephyr_riscv/CMakeLists.txt (hunk 2 above)
Comment (plonajakub) [reply tensorflow/lite/micro/examples/micro_speech/zephyr_riscv/CMakeLists.txt]: Check my update in the latest commit on this

**Pull request merged**